Storage/Storage/Model/WooCommerce.xcdatamodeld/Model 64.xcdatamodel/contents

```diff
@@ -736,7 +736,7 @@
         <attribute name="funding" attributeType="String"/>
         <attribute name="last4" attributeType="String"/>
         <relationship name="charge" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="WCPayCharge" inverseName="cardPresentDetails" inverseEntity="WCPayCharge"/>
-        <relationship name="receipt" optional="YES" maxCount="1" deletionRule="Cascade" destinationEntity="WCPayCardPresentReceiptDetails" inverseName="cardPresentPayment" inverseEntity="WCPayCardPresentReceiptDetails"/>
+        <relationship name="receipt" maxCount="1" deletionRule="Cascade" destinationEntity="WCPayCardPresentReceiptDetails" inverseName="cardPresentPayment" inverseEntity="WCPayCardPresentReceiptDetails"/>
```

Comment:
Since #6082 needs to introduce a new model, it seems like a good opportunity to include this change there.